beetsplug/smartplaylist.py

```diff
@@ -105,17 +105,17 @@ def build_queries(self):
 
             playlist_data = (playlist['name'],)
             try:
-                for key, Model in (('query', Item), ('album_query', Album)):
+                for key, model in (('query', Item), ('album_query', Album)):
```

Comment:
The reason this was capitalized is that it's a class. I get that a naming style checker wouldn't like this because it's also a local variable. If we want to satisfy such a checker, I think we should call it `model_cls` to make it clear what's going on here.

Reply:
`clazz` seems to be a standard around certain parts. What do you think about that? `model_clazz`

Reply:
The reason `clazz` is a thing is for when you want to use `class` alone as a name, but Python reserves the word `class` by itself, so you can't. In any case, however, the convention in Python (and elsewhere in beets) is `cls` (see, for example, the [docs](https://docs.python.org/3/library/functions.html#classmethod) for the `@classmethod` decorator). So let's stick with `model_cls`.